latest/index.bs

```diff
@@ -175,7 +175,261 @@ For this example we assume an image with 5 dimensions and axes called `t,c,z,y,x
+        │   │                 # `.zattrs` MAY contain `"encoding-version"`, which is set to `"0.1.0"` by AnnData.
+        │   │      
+        │   └── col_0         # Each column in the obs table is a 1D zarr array. The rows can be chunked as the user desires.
+        │       ├── .zarray   # However, the obs columns SHOULD be chunked in the same way as the rows in X (if present).
+        │       │
+        │       └─ 0
+        ├── var               # You MAY add a var group container. The var group holds a table of annotations on the columns in X.
+        |   │                 # The rows in var MUST be index-matched to the columns in X (if present). 
+        |   |
+        |   ├── .zattrs       # `.zattrs` MUST contain `"_index"`, which is the name of the column in obs to be used as the index.           
+        |   │                 # `.zattrs` MUST contain `"column-order"`, which is a list of the order of the non-_index columns.
+        |   │                 # `.zattrs` MAY contain `"encoding-type"`, which is set to `"dataframe"` by AnnData.
+        |   │                 # `.zattrs` MAY contain `"encoding-version"`, which is set to `"0.1.0"` by AnnData.
+        |   │
+        |   └── col_0         # Each column in the var table is a 1D zarr array. The rows can be chunked as the user desires.
+        |       ├── .zarray   # However, the var columns SHOULD be chunked in the same way as the columns in X.
+        |       │
+        |       └─ 0
+        |
+        ├── obsm              # You MAY add a obsm group comtainer. The obsm group contains arrays that annotate the rows in X.
+        |   │                 # The rows in each array MUST be index-matched to the rows in X (if present). 
+        |   |
+        │   ├── .zgroup
+        |   |
+        |   ├── .zattrs       # `.zattrs` MAY contain `"encoding-type"`, which is set to `"dict"` by AnnData.           
+        |   │                 # `.zattrs` MAY contain `"encoding-version"`, which is set to `"0.1.0"` by AnnData.
+        |   │
+        │   └── obsm_0        # You MAY add a zarr array for each obsm matrix.
+        |       |             # Each obsm array MUST have the same number of rows as X.
+        |       |             # The rows in each obsm array SHOULD be chunked the same as the rows in X.
+        |       ├── .zarray
+        |       |
+        |       ├── 0.0
+        │       │   ...
+        │       └── n.m
+        |
+        ├── varm              # You MAY add a varm group comtainer. The varm group contains arrays that annotate the columns in X.
+        |   │                 # The rows in each array MUST be index-matched to the columns in X (if present). 
+        |   |
+        │   ├── .zgroup
+        |   |
+        |   ├── .zattrs       # `.zattrs` MAY contain `"encoding-type"`, which is set to `"dict"` by AnnData.           
+        |   │                 # `.zattrs` MAY contain `"encoding-version"`, which is set to `"0.1.0"` by AnnData.
```

Comment:
Starting in anndata 0.8.0, `encoding-type` and `encoding-version` are required.

Since none of these objects have been written yet, I think the spec should be "MUST"

Reply:
Fixed!